src/backend/api/Fusion.Resources.Api/Controllers/Filter/EmulatedUserSupport.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Fusion.Integration.Http.Errors;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Fusion.Integration.Http.Models;
+using Azure.Core;
+using Fusion.AspNetCore.FluentAuthorization;
+using Fusion.Integration;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+
+
+namespace Fusion.Resources.Api.Controllers
+{
+    /// <summary>
```

Comment:
Copied from org service. Mainly done to verify new permission model works as expected for normal managers. 
Can be used by am app to check if user actually has permission to certain endpoints. The attribute should only be used on options endpoint, to prevent data breach.